crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -156,6 +156,18 @@ impl Default for StyledComponentsTransformConfigVc {
     }
 }
 
+/// Configuration options for the custom ecma transform to be applied.
+#[turbo_tasks::value(shared)]
+#[derive(Default, Clone)]
+pub struct CustomEcmaTransformPlugins {
+    /// List of plugins to be applied before the main transform.
+    /// Transform will be applied in the order of the list.
+    pub before: Vec<TransformPluginVc>,
+    /// List of plugins to be applied after the main transform.
+    /// Transform will be applied in the order of the list.
+    pub after: Vec<TransformPluginVc>,
```

Comment:
before and after is a bit vague.

```suggestion
    /// List of plugins to be applied before the main transform.
    /// Transform will be applied in the order of the list.
    pub source_transforms: Vec<TransformPluginVc>,
    /// List of plugins to be applied after the main transform.
    /// Transform will be applied in the order of the list.
    pub output_transforms: Vec<TransformPluginVc>,
```